client/src/app/views/site.coffee

```diff
@@ -0,0 +1,28 @@
+$ = require('jquery')
+Marionette = require('backbone.marionette')
+moment = require('moment')
+SiteModel = require('../models/site')
+template = require('../templates/site.jade')
+
+module.exports = Marionette.ItemView.extend(
+  model: new SiteModel({ parse: true })
+  tagName: 'tr'
+  template: template
+
+  ui:
+    destroy: '.delete'
+
+  events:
+    'click @ui.destroy': 'confirmDelete'
+
+  initialize: ->
+    @model.on('destroy', @remove, this)
```

Comment:
Do we need this?  I thought if the model that was backing an ItemView was destroyed the view was removed automatically for us?